src/Sentry/Laravel/EventHandler.php

```diff
@@ -534,6 +548,58 @@ private function cleanupScopeForOctane(): void
         $this->pushedOctaneScope = false;
     }
 
+    protected function httpClientResponseReceivedHandler(HttpClientEvents\ResponseReceived $event): void
+    {
+        if (!$this->recordHttpClientRequests) {
+            return;
+        }
+
+        $level = Breadcrumb::LEVEL_INFO;
+        if (!$event->response->successful()) {
+            $level = Breadcrumb::LEVEL_ERROR;
+        }
+
+        $fullUri = $this->getFullUri($event->request->url());
+        $partialUri = $this->getPartialUri($fullUri);
```

Comment:
I couldn't come up with a better way to properly generate both versions we need here.
The API of the `GuzzleHttp\Psr7` class is quite narrow.
I also wanted to avoid any string concatenation that will likely be malformed in the end.

Reply:
This seems fine as it is, I'd maybe inline it since the `$partilUri` is only used once, but meh. That's just me, this is fine IMHO!